src/components/ShoppingCart/ShoppingCart.component.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+import ShoppingCartTable from './ShoppingCartTable/ShoppingCartTable.styles';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+
+import { checkoutActionCreators } from 'state/index';
+import { bindActionCreators } from 'redux';
+import { useDispatch } from 'react-redux';
+
+const ShoppingCartComponent = ({className}) => {
+    const { items } = useSelector((state) => state.shoppingCart);
+    const dispatch = useDispatch()
+    const { setCheckoutSummary } = bindActionCreators(checkoutActionCreators, dispatch);
+    const history = useHistory();
+
+    const calculateTotalPrice = (items) => {
```

Comment:
i don't think you need to pass `items` to the function as you have items at the root of your component it is accessible through all your functions
```suggestion
    const calculateTotalPrice = () => {
```